ide/extexecution.process/manifest.mf

```diff
@@ -2,5 +2,5 @@ Manifest-Version: 1.0
 AutoUpdate-Show-In-Client: false
 OpenIDE-Module: org.netbeans.modules.extexecution.process
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/extexecution/process/resources/Bundle.properties
-OpenIDE-Module-Specification-Version: 1.53
+OpenIDE-Module-Specification-Version: 2.0
```

Comment:
I have mixed feelings here. On the one hand there is a massive change in the implementation (dependency bump, drop of native implementation), on the other hand the API did not change in any way. This is still marked a compatible change (same major release version), but to me the specification major bump implies different. I would just count it up to 1.54.

Reply:
this might sound weird, but i did actually use 1.54 first but then I thought @matthiasblaesing will probably suggest to go to 2.x just to be sure, since it bumps the lang level, the position in the lookup and changes the impl at the same time.

I do think staying on 1.54 should be probably fine too. I have to squash anyway so i can change if everyone is ok with it.

Reply:
going to change it to 1.54 and squash, planning to merge once green